bundle.Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=cluster-group-upgrades-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.19.1
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.16.0-ocp
```

Comment:
@nishant-parekh is this the same version you had in your recent PR?

Reply:
/test ci/prow/images

Reply:
@nishant-parekh changed v1.16.0-ocp to 1.19.1 while annotating for the scorecard tests. I think it was an unintentional change (the annotations were intentional, but the version change - not). Please correct me if I'm wrong.
I think we need to have a fixed sdk version per branch. There is a makefile target (make operator-sdk) that downloads the right version.
If we don't make the sdk version fixed, we might have unintentional changes related to auto-generation with every PR.